client/static/coderogue.css

```diff
@@ -58,5 +58,6 @@ span {
 
 #monaco-editor-container {
     min-height: 400px;
+    resize: vertical;
```

Comment:
I guess you previously disabled the resizble (the size gripper at the bottom-right corner) hoping to fix the resizing problems. Now resizing seems to be fine. Should we get back the size gripper so people can drag to expand the height of the editor if they have longer code.